programs/protocol_event/src/lib.rs

```diff
@@ -5,13 +5,9 @@ pub mod state;
 use anchor_lang::prelude::*;
 
 use crate::context::*;
-use crate::state::event_account::{Event, EventStatus, EventType};
+use crate::state::event::*;
+use crate::instructions::CreateEventInfo;
 
-#[cfg(feature = "stable")]
-declare_id!("5qCutonYoeg1aRK31mv4oQYoKdNFMpPaEtDe9nnNQXXf");
-#[cfg(feature = "dev")]
-declare_id!("5qCutonYoeg1aRK31mv4oQYoKdNFMpPaEtDe9nnNQXXf");
-#[cfg(not(any(feature = "stable", feature = "dev")))]
 declare_id!("5qCutonYoeg1aRK31mv4oQYoKdNFMpPaEtDe9nnNQXXf");
```

Comment:
I've removed the extra program id configs for now, as we didn't need them for the product program. Will add them in as/when required